Public/Src/Engine/Dll/Engine.FrontEnd.cs

```diff
@@ -46,6 +47,7 @@ public sealed partial class BuildXLEngine
 
             pipGraph = null;
             IPipGraphBuilder pipGraphBuilder = null;
+            IPipGraphFragmentManager pipGraphFragmentManager = null;

```

Comment:
>pipGraphFragmentManager [](start = 37, length = 23)

Add this logic could be moved into the FrontEndController.

Reply:
done.

---
In reply to: [288668490](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r288668490) [](ancestors = 288668490)